Corp/_config.dist.py

```diff
@@ -0,0 +1,8 @@
+# USAGE STRINGS
+SRP_USAGE = 'The commas are important!\nUsage: *!srp [exact pilot name to receive srp], [zkill lossmail url], ' \
+            '[other comments]*'
+
+# GLOBALS
+SRP_URL = 'SRP FORMRESPONSE URL HERE'
```

Comment:
This could use a comment on how the URL should be formatted and ideally how to get the URL from google apps.